docs/devenv_rhel8.md

```diff
@@ -120,16 +157,17 @@ Verify that the MicroShift is running.
 oc get cs
 oc get pods -A
 ```
-### Stopping MicroShift
-Run the following command to stop the MicroShift and make sure it is shut down by examining its log file.
+
+### Stopping MicroShift Executable
+Run the following command to stop the MicroShift process and make sure it is shut down by examining its log file.
 ```bash
 sudo kill microshift && sleep 3
 tail -3 /tmp/microshift.log 
 ```
 
-Note that this command only stops the MicroShift executable. To perform full cleanup including CRI-O images, run the following script.
+Note that this command only stops the MicroShift executable. To perform full cleanup including `CRI-O` images, run the following script.
```

Comment:
Is it worth adding a note that full cleanup won't remove ovs configuration applied by microshift-ovs-init.service?

Reply:
Good point! Added.